jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -44,6 +44,12 @@
 
 /**
  * <p>A thread pool with a queue of jobs to execute.</p>
+ * <p>The queue of jobs should be unbounded, because critical jobs that have been submitted for
+ * execution cannot be rejected by the queue, which may be only temporarily full due to a job
+ * submission spike.
```

Comment:
```suggestion
 * <p>The queue of jobs should be unbounded, because critical jobs that have been submitted for
 * execution must not be rejected by artificial queue limits.  The queue can be temporarily full
 * due to a job submission spike.
```